src/config.js

```diff
@@ -68,9 +68,27 @@ export default {
         }
       },
       api: {
+        analysisType: {
+          getAllAnalysisTypes: (mid) =>
+            `api/management/${mid.id}/analyses/types`,
+          getAnalysisById: (mid, id) =>
+            `api/management/${mid.id}/analyses/types/${id}`,
+          getAnalysisTypesForCategory: (mid, id) =>
+            `api/management/${mid.id}/analyses/types/categories/${id}`,
+          getAnalysisTypesForCollection: (mid, id) =>
```

Comment:
bear in mind that this greatly decreases test coverage if you have not created tests for these code paths